Automattic/sensei
# Add extensions page tabs

Based on https://github.com/Automattic/sensei/pull/4162

### Changes proposed in this Pull Request

* Introduces the tabs navigation functionality.

### Testing instructions

* Go to the extensions page, and make sure the tabs navigation works properly.
* Also navigate through the setup wizard, and make sure it continues working (the navigation logic was moved to another folder).


### Screenshot / Video


https://user-images.githubusercontent.com/876340/114074097-6271f680-987a-11eb-8e76-fafd1c2889b1.mov
